app/src/main/java/it/niedermann/nextcloud/deck/persistence/sync/adapters/db/dao/StackWidgetModelDao.java

```diff
@@ -0,0 +1,15 @@
+package it.niedermann.nextcloud.deck.persistence.sync.adapters.db.dao;
+
+import androidx.room.Dao;
+import androidx.room.Query;
+import androidx.room.Transaction;
+
+import it.niedermann.nextcloud.deck.model.appwidgets.StackWidgetModel;
+
+@Dao
+public interface StackWidgetModelDao extends GenericDao<StackWidgetModel> {
+
+    @Transaction
```

Comment:
Since you haven't any sub-entities, i think you can omit the `Transaction` annotation